packages/cli-kit/src/public/node/themes/factories.ts

```diff
@@ -8,11 +8,12 @@ interface RemoteThemeJson {
   processing?: boolean
 }
 
-interface RemoteAssetJson {
+export interface RemoteAssetJson {
   key: string
   checksum: string
   attachment: string
   value: string
+  size?: number
```

Comment:
Up for discussion
- creating a new interface / type
- should size be required?